packages/manager/modules/account-sidebar/src/payment-mean/index.scss

```diff
@@ -17,4 +19,8 @@
     font-size: 0.8rem;
     font-weight: bold;
   }
+
+  .oui-icon {
+    color: $hub-title-color;
```

Comment:
Is this different from all  the hub app ? If not it could be globalized

Reply:
yes this color is specific to the payment mean tile